earth_enterprise/rpms/build.gradle

```diff
@@ -402,16 +402,84 @@ if (project.hasProperty('buildOpenGee')) {
     openGeeCommonRpm.dependsOn stageOpenGeeInstall
 }
 
-
-task openGeeCommonDeb (type: Deb) {
+task openGeeCommonDeb (type: GeeDeb, dependsOn: openGeeSharedFiles) {
     packageName = 'opengee-common'
+    release = "1.${rpmPlatformString}"
+    version = openGeeVersion
+    user = 'root'
+    permissionGroup = 'root'
+    packageGroup = 'Application/Productivity'
+    summary = 'Third-party libraries bundled with Open GEE'
+    // The package build failed when packageDescription was copied here from openGeeCommonRpm, so it is omitted for now
+    license =
+        'ASL 2.0 and ' +
+        'BSD and ' +
+        'GPLv2+ and ' +
+        'Psycopg2 GPL 3+ and ' +
+        'LGPLv2 and ' +
+        'MIT and ' +
+        'MPLv1.1 and ' +
+        'PSF 2 and ' +
+        'Python Imaging Library 1.1.7 and ' +
+        'mgrs 1.1.0 and mm 1.4.2 and ' +
+        'OpenLDAP and ' +
+        'OpenSSL and ' +
+        'Skia 5.1.2'
 
-    from(stagedInstallDir_common) {
+    setArch('amd64')	// if this is not specified, the architecture is x86_64, and dpkg won't install without the --force-architecture flag
+
+    requires('alien')
```

Comment:
The following packages contain build tools, and should not be necessary at runtime: `alien`, `autoconf`, `automake`, `doxygen`, `python-git`, `shunit2`, `swig`.

Reply:
Thanks. I'll remove those dependencies.